### DISCUSSION TITLE
BTCPay horizontal scaling

### DISCUSSION THREAD

**Discussion by mobycrypt** [btcpayserver/btcpayserver#2874] [General]
Hello. Is BTCPay ready to be horiztonally scalled - **running multiple instances of btcpay on single database**? What would be the restrictions or side effects of scaling btcpay?

I'm asking because I design the service where I assume there can be quite big load and I planned to scale it horizontally - run multiple instances of btcpay and use nginx for handling requests load. I assume single NBXplorer and single full node can be used by many btcpay instances.

I see potential problems with payments scanning and webhooks - every instance would notify about received payment, because every instance is scanning the same invoices.

Are there any other known restrictions for even trying horizontally scale BTCPay?

Thank you in advance for all info.

**Comment by Bobby1332** [comment]
I m trying to pay from my account if ivr not depleted it any advice plrrade much APORECIATE

**Comment by Bobby1332** [comment]
Yes of course ill gladly kick in

**Comment by Bobby1332** [comment]
I. Do not know them smh

**Comment by Bobby1332** [comment]
Some help ??.
